File: mobile/src/main/java/uk/co/appsbystudio/geoshare/utils/ui/SeekBarTextIndicator.java
package uk.co.appsbystudio.geoshare.utils.ui;

import android.content.Context;
import android.graphics.drawable.Drawable;
import android.support.v7.widget.AppCompatSeekBar;
import android.util.AttributeSet;

public class SeekBarTextIndicator extends AppCompatSeekBar {

    public SeekBarTextIndicator(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    private Drawable mThumb;

    @Override
    public void setThumb(Drawable thumb) {
        super.setThumb(thumb);
        mThumb = thumb;
    }

    public void getSeekBarThumb() {
    }
}

File: mobile/src/main/java/uk/co/appsbystudio/geoshare/utils/ui/SeekBarTextIndicator.kt
package uk.co.appsbystudio.geoshare.utils.ui

import android.content.Context
import android.graphics.drawable.Drawable
import android.support.v7.widget.AppCompatSeekBar
import android.util.AttributeSet

class SeekBarTextIndicator(context: Context, attrs: AttributeSet) : AppCompatSeekBar(context, attrs) {

    private var mThumb: Drawable? = null

    override fun setThumb(thumb: Drawable) {
        super.setThumb(thumb)
        mThumb = thumb
    }
}
